extensions/applicationinsights-react-native/src/ReactNativePlugin.ts

```diff
@@ -100,10 +141,23 @@ export class ReactNativePlugin implements ITelemetryPlugin {
         }
     }
 
+    /**
+     * Automatically collects unhandled JS exceptions and native exceptions
+     */
+    private _setExceptionHandlers() {
+        setJSExceptionHandler((error, isFatal) => {
+            this._trackException({exception: error});
```

Comment:
What does `isFatal` mean? Will the app crash and telemetry be dropped? Would it require a flush after trackException?

Reply:
It's a self defined param of customized error handler ([example](`react-native-device-info`)), I think we don't need it here, removed it. thanks 👍